src/kernel/src/syscalls/mod.rs

```diff
@@ -302,15 +311,23 @@ impl Syscalls {
                 let v2 = read::<u32>(req.add(8) as _);
                 let value = read::<i32>(req.add(12) as _);
 
+                info!(
+                    "Attempting to set registry with v1: {}, v2: {}, value: {}.",
+                    v1, v2, value
+                );
                 self.regmgr.decode_key(v1, v2, td.cred(), 2).and_then(|k| {
-                    info!("Setting registry {k} to {value}.");
+                    info!("Setting registry key {} to value {}.", k, value);
                     self.regmgr.set_int(k, value)
                 })
             }
             0x19 => {
                 let v1 = read::<u64>(req as _);
                 let v2 = read::<u32>(req.add(8) as _);
 
+                info!(
+                    "Attempting to decode registry key with v1: {}, v2: {}.",
+                    v1, v2
+                );
```

Comment:
Same here.

Reply:
Why so? 0x18 and 0x19 use separate info! prompts because they use different variables... So why is 0x18 fine, but 0x19 not?

Reply:
Because 0x18 is a write operation while 0x19 is a read operation. That mean 0x18 will alter the state of the game, which make log is useful.

Reply:
Done